### PR TITLE
OADP-5246: Slight update to OADP / ROSA documentation

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [OADP-5246](https://issues.redhat.com/browse/OADP-5246)

### Version(s):

* OCP 4.12 → branch/enterprise-4.12
* OCP 4.13 → branch/enterprise-4.13
 * OCP 4.14 → branch/enterprise-4.14
* OCP 4.15 → branch/enterprise-4.15
* OCP 4.16 → branch/enterprise-4.16
* OCP 4.17 → branch/enterprise-4.17
* OCP 4.18 → branch/enterprise-4.18

### Link to docs preview:

* [Installing the OADP Operator and providing the IAM role](https://85241--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.html#installing-oadp-rosa-sts_oadp-rosa-backing-up-applications)

### QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by anarnold97** [comment]
@weshayutin - if you are ok with the wording, i will copy in @bhardesty  to handle from the ROSA docs side and see if he wants me to change this warning to a snippet so they can include directly

**Review by weshayutin — APPROVED** [review]
/LGTM

**Comment by weshayutin** [comment]
THANK YOU @anarnold97

**Comment by bhardesty** [comment]
@anarnold97 Thanks for the heads-up. Thankfully, this module appears to be reused as-is in the ROSA docs, so we should be good (https://github.com/openshift/openshift-docs/blob/main/rosa_backing_up_and_restoring_applications/backing-up-applications.adoc?plain=1#L18).

**Review by stillalearner — APPROVED** [review]
lgtm

File: modules/installing-oadp-rosa-sts.adoc [hunk 1]
```diff
@@ -11,7 +11,17 @@ AWS Security Token Service (AWS STS) is a global web service that provides short
 
 [IMPORTANT]
 ====
-Restic and Kopia are not supported in the OADP on ROSA with {aws-short} {sts-short} environment. Verify that the Restic and Kopia node agent is disabled.
+Restic is unsupported.
+
+Kopia file system backup (FSB) is supported when backing up file systems that do not have Container Storage Interface (CSI) snapshotting support.
+
+Examples include:
```
Comment (skrthomas) [inline modules/installing-oadp-rosa-sts.adoc:18]:
```suggestion
Example file systems include the following:
```

**Review by skrthomas — COMMENTED** [review]
Just a small nit to introduce the list with a complete sentence. Looks good otherwise 👍

**Comment by anarnold97** [comment]
@skrthomas - thanks

File: modules/installing-oadp-rosa-sts.adoc [hunk 2]
```diff
@@ -11,7 +11,17 @@ AWS Security Token Service (AWS STS) is a global web service that provides short
 
 [IMPORTANT]
 ====
-Restic and Kopia are not supported in the OADP on ROSA with {aws-short} {sts-short} environment. Verify that the Restic and Kopia node agent is disabled.
+Restic is unsupported.
+
+Kopia file system backup (FSB) is supported when backing up file systems that do not have Container Storage Interface (CSI) snapshotting support.
+
+Example file systems include the following:
```
Comment (michaelryanpeter) [inline modules/installing-oadp-rosa-sts.adoc:18]: Nit. This should be rephrased [to conform with the ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#following). As a lead-in for a list or procedure, _following_ should be used as an adjective and not a noun:
```suggestion
Supported file systems include the following examples:
```
That said, this isn't a merge blocker. Please take care of this in the future in a follow up PR.

**Review by michaelryanpeter — COMMENTED** [review]
One small terminology nit that is not enough to block a merge.

**Pull request merged**